by-language/java-jooq/src/main/java/io/crate/demo/jooq/Application.java

```diff
@@ -91,7 +90,7 @@ public void exampleWithGeneratedCode() throws IOException, SQLException {
         DSLContext db = getDSLContext();
 
         // Create table.
-        String bootstrap_sql = Tools.readTextFile("bootstrap.sql");
+        String bootstrap_sql = Tools.readTextFile("author.sql");
         db.query(bootstrap_sql).execute();
```

Comment:
Maybe `executeBatch` should be used here, to accomodate hosting multiple SQL statements within the same file. Discovered via [1].

/cc @seut 

[1] https://github.com/cockroachdb/examples-orms/blob/master/java/jooq/src/main/java/com/cockroachlabs/util/Utils.java#L33-L36

Reply:
Apparently, it wasn't needed. I reverted back to using a single `bootstrap.sql` file, and it worked well. It must have been a glitch of some kind.